client/components/payment-activity/payment-activity-data.tsx

```diff
@@ -230,11 +249,19 @@ const PaymentActivityDataComponent: React.FC< Props > = ( {
 							) }
 							content={ interpolateComponents( {
 								mixedString: __(
-									'{{strong}}Fees{{/strong}} includes fees on payments as well as disputes.',
+									'{{strong}}Fees{{/strong}} includes all types of fees charged by WooPayments. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
```

Comment:
The `Learn more` links are useful 🚀 

@souravdebnath1986 should we add a `Learn more` link to the `Disputes` tile tooltip also? Seeing a number in this tile might be a merchant's first experience of a dispute!

Reply:
I agree. Disputes could also have a `Learn more` link to the [support page](https://woocommerce.com/document/woopayments/fraud-and-disputes/) for merchant education purposes. cc: @aheckler